packages/react/src/components/Button/Button.js

```diff
@@ -205,7 +205,6 @@ Button.defaultProps = {
   tabIndex: 0,
   type: 'button',
   disabled: false,
-  small: false,
```

Comment:
Maybe we could add the deprecate helper here and just make sure it doesn't get spread onto any underlying nodes since this was deprecated in v9?

Reply:
but do we still need the default value if we've already removed `.isRequired` for this prop?

Reply:
Totally fair!